tox.ini

```diff
@@ -35,7 +35,9 @@ deps =
 commands = flake8 flake8_import_order/
 
 [testenv:setuppy]
-deps = docutils
+deps =
+    docutils
+    Pygments
```

Comment:
@sigmavirus24 Because I added `.. code-block:: python` directive to README.rst.  Docutils warns if Pygments is not installed.